src/document/HolonomicPathStore.ts

```diff
@@ -224,6 +231,10 @@ export const HolonomicPathStore = types
   })
   .actions((self) => {
     return {
+      setVisibleWaypoints(start: number | number[]) {
```

Comment:
I was kind of just taking the code from `ConstraintsConfigPanel.tsx` where on the slider's `onChange` they have this: `(e, value: number | number[])` as the parameters that the lambda (? idk what the word is in javascript) takes in and then uses to set where the constraint applies to. I think that's just what a slider returns since it can be on just 1 number or on multiple.

I have read your next suggestion though and will implement that as a solution as it makes sense. Thanks